contrib/node/README.md

```diff
@@ -1,53 +1,81 @@
-# Plugin to support JS/Node
+# Plugin to support JavaScript and Node.js
 
-This is a plugin to integrate Node/NPM with Pants.
+This is a plugin to integrate Node.js with Pants.
 
 # Targets
 
-You can specify node_remote_module targets referring to remote 3rd party Node modules, and
-you can specify node_module targets defining projects in the source tree.
+Node.js targets in pants are designed to work with existing Node.js package management tools. A node_module target will work with a package.json
+source file in the same directory, combining the BUILD dependencies with the additional fields from the source package.json to form the complete
+package.json used for tasks on the target.
```

Comment:
I found this paragraph a little confusing to follow. I took a crack at rewording it. What do you think?

```
Node.js targets in Pants are designed to work with existing Node.js package management tools and
their configuration files. The node_module target type uses a package.json file in the same
directory as its definition. It combines dependencies declared on the target in the BUILD
file with the original package.json to form a complete package.json that can be used to run tasks
on the target through Pants.
```